saleor/tests/e2e/product/utils/product_query.py

```diff
@@ -0,0 +1,84 @@
+from ...utils import get_graphql_content
+
+PRODUCT_QUERY = """
+query Product($id: ID!, $channel: String) {
+  product(id: $id, channel: $channel) {
+    id
+    name
+    attributes {
+      attribute {
+        id
+      }
+      values {
+        name
+      }
+    }
+    pricing {
+      onSale
+    }
+    variants {
+      id
+      name
+      pricing {
+        onSale
+        discount {
+          gross {
+            amount
+          }
+        }
+        priceUndiscounted {
+          gross {
+            amount
+          }
+        }
+      }
+      attributes {
+        attribute {
+          id
+        }
+        values {
+          name
+        }
+      }
+      channelListings {
+        channel {
+          id
+        }
+        price {
+          amount
+          currency
+        }
+      }
+      stocks {
+        warehouse {
+          id
+        }
+        quantity
+      }
+    }
+  }
+}
+"""
+
+
+def get_product(
+    staff_api_client,
```

Comment:
`api.client` - Same here since not only staff should be able to get product info